docs/userguide.md

```diff
@@ -0,0 +1,151 @@
+# User Guide
+
+## What is the cluster registry?
+
+The cluster registry is a Kubernetes-style API and API server that provides an
+endpoint for interacting with a list of clusters and associated metadata. If it
+helps, you can think of the cluster registry as a hosted kubeconfig file.
+However, since it's a Kubernetes API server, the cluster registry allows custom
+annotations and filtering across labels, can be used with `kubectl` and
+Kubernetes-style generated client libraries, and supports having controllers
+watch for updates.
+
+## Deploying a cluster registry
+
+The `crinit` tool allows you to deploy a cluster registry into an existing
+Kubernetes cluster. Run the following commands to download the latest nightly
+build of the tool:
+
+> Currently the tool is only released for Linux 64-bit. You'll need to built it
+> yourself if you want to use it on a different platform. See
+> [the development docs](development.md#Building-crinit).
+
+```sh
+PACKAGE=client
+LATEST=$(curl https://storage.googleapis.com/crreleases/nightly/latest)
+curl -O http://storage.googleapis.com/crreleases/nightly/$LATEST/clusterregistry-$PACKAGE.tar.gz
+tar xzf clusterregistry-$PACKAGE.tar.gz
+```
+
+### Standalone
+
+You can deploy the cluster registry as a standalone API server like so:
+
+```sh
+./crinit standalone <cluster_registry_context> --host-cluster-context=<your_cluster_context>
```

Comment:
You probably need to `cd` into the extracted directory at some point? Without `cd` the flow kind of breaks for people following along?

Reply:
It's best to call the positional argument the cluster registry instance name. It has implications far beyond a context name in the local kubeconfig file. For example, it is used for the deployment and service names so the name should conform to those rules etc.

Reply:
There is no extracted directory: the command is extracted into the current working directory.

The point about the positional argument makes a lot of sense. Fixed.

Reply:
> There is no extracted directory: the command is extracted into the current working directory.

I see Ok. Thanks for the explanation.